datasets/san_francisco_bikeshare_trips/bikeshare_trips/bikeshare_trips_dag.py

```diff
@@ -76,7 +76,7 @@
             "data/san_francisco_bikeshare_trips/bikeshare_trips/data_output.csv"
         ],
         source_format="CSV",
-        destination_project_dataset_table="san_francisco_bikeshare_trips.bikeshare_trips",
+        destination_project_dataset_table="{{ var.json.san_francisco_bikeshare_trips.container_registry.bikeshare_trips_destination_table }}",
```

Comment:
Please change to san_francisco_bikeshare as well

Reply:
@adlersantos Please see my last comment